### ISSUE TITLE
Both .bolt.yml and bolt.yml installed in a Composer create-project install

### ISSUE THREAD

**Issue by GawainLynch** [bolt/bolt#5894]
Speaking of testing :wink:

**Comment by PhillippOhlandt** [comment]
Maybe because it's still here?:
https://github.com/bolt/composer-install/blob/master/.bolt.yml

**Comment by GawainLynch** [comment]
Yes, and *should* be there.

**Comment by bobdenotter** [comment]
@GawainLynch Same issue as this: https://github.com/bolt/composer-install

**Comment by GawainLynch** [comment]
Ah, yes … but we need to adjust core's behaviour IMHO.

And still not satisfied that our approach is correct here, especially given the subtle split of packaging, e.g. `.gitignore` 

Let's hash this over tonight at the meeting, it's a bit of a full day for you and I, but I don't want to hold up release :smile:

**Comment by bobdenotter** [comment]
I'm not saying that my approach was best.. Just linking them together, as they're effectively the same problem.

**Comment by bobdenotter** [comment]
BTW, I'm stripping the _extra_ `.bolt.yml` out during packaging.

**Comment by GawainLynch** [comment]
> I'm not saying that my approach was best.. Just linking them together, as they're effectively the same problem.

Oh oh oh oh… If I came across critical, I communicated badly.

What I was wanting to express was that there seems to be a mixed, and not well defined, approach that is _almost_ over that hurdle but we just need to step back and ensure we're consistent.

> BTW, I'm stripping the extra .bolt.yml out during packaging.

Yep, good to know. More worried about those of us that install this way and get bitten… driving up the support effort needlessly.
